frontend/packages/data-portal/app/components/Run/ViewerPage.tsx

```diff
@@ -34,7 +40,64 @@ import { CryoETHomeLink } from '../Layout/CryoETHomeLink'
 import { Tooltip } from '../Tooltip'
 import { NeuroglancerBanner } from './NeuroglancerBanner'
 import { getTutorialSteps } from './steps'
-import Tour from './Tour'
+import { Tour } from './Tour'
+
+interface WatchableBoolean {
```

Comment:
@seankmartin I think it would be better to move the interface definitions to the neuroglancer wrapper, this would avoid to have mixed concerns between the view in itself, and technical types for the neuroglancer states. What do you think?